### PR TITLE
Adding documentation for test tool configuration scripts

### PR DESCRIPTION
**Pull request by burzum619**

### CODE REVIEW

File: _deployment/configuration/universal_tools.md [hunk 1]
```diff
@@ -20,32 +20,50 @@ previous year. The three tables of concern are:
...
-WHERE context = 'SBAC-IRP-Mathematics-7';
-```
-```
-INSERT INTO configs.client_tooldependencies
-SELECT * FROM configs.client_tooldependencies
-WHERE context = 'SBAC-IRP-Mathematics-7';
-```
-3. Flush the redis cache
+
+## Test Tool Configuration Stored Procedures
```
Comment (millerts) [inline _deployment/configuration/universal_tools.md]: Can we link to the document that has all the possible accommodation codes?

Comment (millerts) [reply _deployment/configuration/universal_tools.md]: Oh, so this enables all accommodations rather than specific ones?

Comment (burzum619) [reply _deployment/configuration/universal_tools.md]: That's correct Todd. The "General Tools" inserts all "universal tools" and a few accommodations and designated supports. The `configs.client_testtooltype` table does cascaded deletes to the `configs.client_testtool` and `configs.client_tooldependencies` tables, so if we wanted to remove a specific tool type, we'd do a `DELETE FROM configs.client_testtooltype WHERE context = '<assessment id>' AND clientname = '<client>' AND toolname = '<tool name>';`. That would remove all tool options and tool dependencies.

Comment (burzum619) [reply _deployment/configuration/universal_tools.md]: The ISAAP codes link is in the "## Configuring Universal Tools/Designated Supports/Accommodations for Specific Assessments and Segments" already. Do you want me to include it in this other segment as well?

Comment (millerts) [reply _deployment/configuration/universal_tools.md]: No, I didn't realize that was already there.  I pulled this down and looked at it in MacDown which is much easier.  My bad.

I didn't see a part in the documentation where it is explained how to remove the tools. Basically what you have in the comment above.  Can you add a section for that as well?  I think the common operating procedure will be to run the new stored procedures and then remove the ones that aren't needed.  Unless I missed that part in the documentation?

**Comment by burzum619** [comment]
Closing PR - Some additional documentation changes are necessary, and we will be opening a new PR with some additional changes by @jeffjohnson9046

**Pull request closed without merging**